### ISSUE TITLE
Importing MassiveCore Factions into FactionsOne

### ISSUE THREAD

**Issue by TheHardestBunnyAlive** [DRE2N/FactionsOne#35]
Hey, I was wondering if it would be possible to import the MassiveCore Factions into FactionsOne. Is there a folder where I could drag the current factions made with MassiveCore and put them into a folder in FactionsOne?
Thanks,
plisov

**Comment by Sataniel98** [comment]
No, FactionsOne and MassiveCore's Factions have few to nothing to do with each other technically, similar to Towny and Factions.

**Comment by TheHardestBunnyAlive** [comment]
Ah thanks. Also, how do I hook into your plugin? I'm trying to implement it into one of my plugins.
